tests/integration/test_aiohttp.py

```diff
@@ -1,3 +1,4 @@
+import multidict
```

Comment:
I think here we'll have to use `pytest.importorskip('multidict')` to avoid all those CI builds failures. We had to do th same with aiohttp (on line 3)